Makefile

```diff
@@ -20,10 +20,10 @@ prune:
 	@docker-compose down -v
 
 ps:
-	@docker ps --filter name='$(PROJECT_NAME)*'
+	@docker ps --filter name="$(PROJECT_NAME)*"
 
 shell:
-	docker exec -ti $(shell docker ps --filter name='$(PROJECT_NAME)_php' --format "{{ .ID }}") sh
+	docker exec -ti "$(shell docker ps --filter name="$(PROJECT_NAME)_php" --format "{{ .ID }}") sh"
```

Comment:
this command was working before, I don't know why you added the quotation marks but it is not working :( 
As it is now with the quotation mark, the command now it is confused and wrapping things different that you are expecting, for example, you are saying that this is one argument:
"$(shell docker ps --filter name="
and this is another one:
" --format "

Reply:
Hi @ivonne-gov I ran the code through a linter and that was the recommended update. I thought I had tested all the make commands, but maybe I missed this one.

It makes sense what you are explaining though.  Let me go in and update back to our previous state and push up the fix.